src/baguetteBox.js

```diff
@@ -595,9 +636,14 @@
         });
     }
 
-    function bind(element, event, callback) {
+    // Modified to allow useCapture option. Needed for trapFocusHandler and
+    // to conform to wai-aria accessibility guidelines on modals/popup
+    function bind(element, event, callback, useCapture) {
+        if (useCapture === undefined) {
+            useCapture = false;
+        }
```

Comment:
We can get rid of this comment now. Also the `if` statement is unnecessary - `addEventListener` will handle `undefined` gracefully.